refocus-bdk-server.js

```diff
@@ -188,46 +204,87 @@ module.exports = (config) => {
 
     const socket = io.connect(connectUrl, opts);
 
-    socket.on(initalizeEventName, (data) => {
+    socket.on(initalizeEventName, async (data) => {
       const eventData = JSON.parse(data);
       const room = eventData[initalizeEventName];
+      const { id, updatedAt } = room;
+      const hasAlreadyBeenConsumed = cache &&
+        await cache.hasBeenConsumed(id, updatedAt, initalizeEventName);
+      if (hasAlreadyBeenConsumed) {
```

Comment:
could inline this 'if () return';

Reply:
for sure - .eslint doesn't like this but can change that too

Reply:
yep, change .eslint :)